src/app/large-display/large-display.component.ts

```diff
@@ -0,0 +1,28 @@
+import { Component, Inject, OnInit, ViewChild, HostListener } from '@angular/core';
+import { TimerBarComponent } from 'src/app/timer-bar/timer-bar.component'
+
+@Component({
+  selector: 'app-large-display',
+  templateUrl: './large-display.component.html',
+  styleUrls: ['./large-display.component.scss']
+})
+export class LargeDisplayComponent implements OnInit {
+
+  componentHeight: number;
+
+  @ViewChild('timerBar', {static: true}) timerBar: TimerBarComponent;
```

Comment:
You should generally avoid accessing the methods of a component this way. The timer bar component shouldn't handle the timing itself anyway and should be passed in a time to show as an input.